guava/pom.xml

```diff
@@ -38,12 +38,13 @@ Guava (https://github.com/google/guava) types (currently mostly just collection
     <!-- 28-Feb-2023, tatu: 2.15 raises baseline from 21.x to 23.x -->
     <!-- 01-May-2023, tatu: 2.16 raises baseline from 23.x to 25.x -->
     <version.guava>25.1-jre</version.guava>
+    <version.guava.osgi-base>25.1</version.guava.osgi-base>
 
     <javac.src.version>1.8</javac.src.version>
     <javac.target.version>1.8</javac.target.version>
 
     <!-- 31-Jul-2020, tatu: remove upper limit for Jackson 2.12; tested up to 30.x -->
-    <version.guava.osgi>[${version.guava}.0,99)</version.guava.osgi>
+    <version.guava.osgi>[${version.guava.osgi-base}, 99)</version.guava.osgi>
```

Comment:
A space has been added before `99`- I am not sure if this is allowed in an OSGi version range selector. I would recommend to omit the space.

Reply:
Thanks, that was unintentional -- I suspect it should be ok but better safe than sorry so will remove